docs/modules/ROOT/pages/sentinel.adoc

```diff
@@ -141,7 +141,11 @@ Transaction Conditions can refer to the following properties
 
 * *from* is the _from_ address for the transaction
 
-* *gasPrice* is the price of gas sent in the transaction
+* *gasPrice* is the price of gas sent in the transaction. In EIP1559 transactions, it's usually the same as `maxFeePerGas`.
```

Comment:
Is it the case in 1559s, or is it the actual price paid? If so, let's reprhase to "it's equal to or below the `maxFeePerGas`".

Reply:
There is no spec for it. I tried with some RPC calls and every one of them was returning the same as `maxFeePerGas` but I don't want to commit to that since I didn't find any official spec.

Updating!

Reply:
See [here](https://hackmd.io/@timbeiko/1559-json-rpc#API-Changes) for the spec (it's pretty hard to find!)

Reply:
Gotcha, I recall seeing that from the research on EIP1559 docs. The tests I ran were in Polygon where the baseFee is so low that the effective price is always `maxFeePerGas`

This is a confirming example
```
curl -X POST \
    -H "Content-Type: application/json" \
    -d '{"jsonrpc":"2.0","method":"eth_getTransactionByHash","params": ["0x30a26da2c1f947bd0652b3086262fe907fa11f6c71ba7664dab6200ca6da5562"],"id":1}' https://mainnet.infura.io/v3/<INFURA_ID>
```

Thanks!